settings/dev.settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ROOT_URL": "",
-  "MAIL_URL": "",
+  "MAIL_URL": "smtp://abdulfataiaka:fattyolaide22olaidefatty@smtp.gmail.com:465",
```

Comment:
@temilaj could you comment on the use of a personal email? Or is this fine for the project?